app/design/frontend/Magebit/Learning/Magento_Catalog/templates/product/view/addtocart.phtml

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @copyright Copyright (c) 2024 Magebit (https://magebit.com)
+ * @author    Magebit <info@magebit.com>
+ * @license   GNU General Public License ("GPL") v3.0
+ */
+/** @var Magento\Catalog\Block\Product\View $block */
+/** @var \Magento\Catalog\Model\Product $product */
+/** @var \Magento\CatalogInventory\Api\StockRegistryInterface $stockRegistry */
+$product = $block->getProduct();
+$objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+$stockRegistry = $objectManager->get(\Magento\CatalogInventory\Api\StockRegistryInterface::class);
+$stockItem = $stockRegistry->getStockItem($product->getId());
+$qty = $stockItem->getQty();
+
+?>
+<form action="<?= $block->getAddToCartUrl($product) ?>" method="post">
+    <div class="addtocart-product-info">
+        <div class="qty-container">
+            <button type="button" class="qty-btn" id="sub" data-bind="click: function() { subtract(); }">-</button>
```

Comment:
it is better to use icons instead of "-" or "+", but for learning I will allow it. Next time use icons